1.3.7 版本构建出错

```bash
This dependency was not found:

* /home/admin/workspace/bd-platform/node_modules/_umi-request@1.3.7@umi-request in ./src/.umi-production/plugin-request/request.ts

To install it, you can run: npm install --save /home/admin/workspace/bd-platform/node_modules/_umi-request@1.3.7@umi-request
Child
    
    ERROR in ./src/.umi-production/plugin-request/request.ts
    Module not found: Error: Can't resolve '/home/admin/workspace/bd-platform/node_modules/_umi-request@1.3.7@umi-request' in '/home/admin/workspace/bd-platform/src/.umi-production/plugin-request'
     @ ./src/.umi-production/plugin-request/request.ts 10:0-131 75:26-32
     @ ./src/.umi-production/core/umiExports.ts
     @ ./src/.umi-production/plugin-dva/dva.ts
     @ ./src/.umi-production/plugin-dva/runtime.tsx
     @ ./src/.umi-production/core/pluginRegister.ts
     @ ./src/.umi-production/umi.ts
     @ multi ./src/.umi-production/umi.ts
```

--- a/.fatherrc.js
+++ b/.fatherrc.js
@@ -1,3 +1,4 @@
 export default {
   esm: 'rollup',
+  cjs: 'rollup',
 };

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "version": "1.3.7",
   "description": "A request tool based on fetch.",
   "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "module": "dist/index.esm.js",
   "repository": {
     "type": "git",
     "url": "git@github.com:umijs/umi-request.git"
@@ -36,6 +36,7 @@
     "iconv-lite": "^0.4.24",
     "jest": "^23.5.0",
     "np": "5.0.2",
+    "prettier": "^2.3.0",
     "query-string": "^6.9.0",
     "typescript": "^3.0.3",
     "umi": "^2.8.15",

